acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ProcessBesuNodeRunner.java

```diff
@@ -105,7 +106,21 @@ public void startNode(final BesuNode node) {
       params.add("--privacy-precompiled-address");
       params.add(String.valueOf(node.getPrivacyParameters().getPrivacyAddress()));
       params.add("--privacy-marker-transaction-signing-key-file");
-      params.add(node.homeDirectory().resolve("key").toString());
+      node.getPrivacyParameters()
+          .getSigningKeyPair()
+          .ifPresentOrElse(
+              keyPair -> {
+                try {
+                  File privKeyFile = File.createTempFile("privKey", ".tmp");
```

Comment:
do you need to flag the tempFIle as "deleteOnExit"?

Reply:
Yep, great idea.  👍